src/extractors/custom/clinicaltrials.gov/index.js

```diff
@@ -0,0 +1,29 @@
+export const ClinicaltrialsGovExtractor = {
```

Comment:
There seems to be some changes made on the website's structure since you have created this custom extractor, most of the class names seem to be prefixed by `tr-`, so some of the selectors here no longer work.
Could you please update the fixture and selectors accordingly?

Reply:
Updated 👍